frontend/src/pages/dashboard/components/sidebar/sidebar.tsx

```diff
@@ -23,30 +32,44 @@ const Sidebar: React.FC<Properties> = ({ isCollapsed = false }: Properties) => {
     [navigate],
   );
 
+  const renderTabs = useCallback(() => {
+    const tabs = getTabs();
+
+    return tabs.map((tab) => (
+      <li
+        className={getValidClassNames(
+          styles.item,
+          checkActiveTab(location.pathname, tab.path) && styles.active,
+        )}
+        key={tab.name}
+      >
+        <Button
+          {...(!isCollapsed && { label: tab.name })}
+          className={[
+            'h5',
+            styles.btn,
+            {
+              [styles.active]: checkActiveTab(location.pathname, tab.name),
```

Comment:
You can't use getValidClassNames to pass classes to the button? I'm not sure, that's why I'm asking. I see one approach on `li` and another here.